🐛 (games) Adding Handling Input Changes page

emilieroberts opened the pull request:
Adding Handling Input Changes page

Adding page about how apps can handle input device changes.

Commit:
🐛 (games) Adding Handling Input Changes page

Adding Handling Input Changes page

Commit:
🐛 (games) Adding Handling Input Changes page

Adding Handling Input Changes page

emilieroberts committed:
Delete package-lock.json

Should not have been included.

emilieroberts committed:
Delete package.json

Should not have been included.

Snugug reviewed (changes requested):
Please add `package.json` and `package-lock.json` back in

Snugug commented on pages/en/games/handling-input-changes.md:
```diff
@@ -0,0 +1,357 @@
+
+Begin by identifying all of the **input states** your game might be running in and then listing all the parameters that you would like to change in each state.
+
+<table>
```
Please write this table [using Markdown](https://www.markdownguide.org/extended-syntax/). You can use this [table generator](https://www.tablesgenerator.com/markdown_tables) as a starting place if you'd like

Snugug commented on pages/en/games/handling-input-changes.md:
```diff
@@ -0,0 +1,357 @@
+
+![Prioritized state machine - touchscreen, keyboard/mouse, game controller](/images/games/optimizing-games-input/input-state-machine.png)
+
+<table>
```
Same with this table

Snugug commented on pages/en/games/handling-input-changes.md:
````diff
@@ -0,0 +1,357 @@
+```
+
+!!! aside.message--note
+**Note**: for KeyEvents, you can choose to use either [onKeyDown()](<https://developer.android.com/reference/android/app/Activity#onKeyDown(int,%20android.view.KeyEvent)>) or [onKeyUp()](<https://developer.android.com/reference/android/app/Activity#onKeyUp(int,%20android.view.KeyEvent)>). Here, [onKeyDown()](<https://developer.android.com/reference/android/app/Activity#onKeyDown(int,%20android.vi ...
````
Can you please put these function calls in code ticks? Thanks!

Snugug reviewed (changes requested):
A couple of small formatting items I noticed as well

Commit:
Merge branch 'input' of https://github.com/chromeos/chromeos.dev into input

Commit:
🐛 (games) Adding Handling Input Changes page

Adding Handling Input Changes page

commit 0934d39bf97d820baa5f6625e33d7a6984990623

Snugug committed:
🐛 (code) Add package.json and lock back in

fuentespatrick commented:
Merging since @Snugug already reviewed and feedback was addressed
